Alpine.js

```diff
@@ -94,15 +102,15 @@ window.Alpine = {
     },
 
     addListenerToElement(el) {
-        Array.from(el.attributes).forEach((attribute) => {
+        for (const attribute of el.attributes) {
             if (!attribute.name.startsWith("@")) return;
 
-            let event = attribute.name.substring(1);
+            const event = attribute.name.substring(1);
 
             el.addEventListener(event, () => {
                 eval(`with(this.data) { ${attribute.value} }`);
             });
-        });
+        }
```

Comment:
Why did you replace `Array.from()`  with `for...of` ?

Reply:
I think for. . of syntax is more concise and read-only here. So I don't think it's necessary to copy new memory to prevent mutate。This project is excellent, if you think Array.from() is better, I will respect your decision and correct the re-propose PR